src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/BindableTypeProvidersGenerationTask.cs

```diff
@@ -222,7 +222,7 @@ private void GenerateProviderTable(IEnumerable<INamedTypeSymbol> q, IndentedStri
 							using (writer.BlockInvariant(@"if(!_knownMissingTypes.Contains(type))"))
```

Comment:
```suggestion
							using (writer.BlockInvariant(@"if(!_knownMissingTypes.Contains(type) && !type.IsGenericType)"))
```

This change could the help with the incorrect message. Could you validate that it helps for the generic case ?